src/Components/Main/Pages/ExamplesPage/ExampleCharts/ExampleCharts.tsx

```diff
@@ -62,11 +53,13 @@ export const ExampleCharts: FC<PropsType> = ({
   const { t } = useTranslation();
 
   const exampleFirstInitialValue =
-    languageMode === Language.Ru ? exampleFirstInitialValueRu : exampleFirstInitialValueEn;
+    languageMode === Language.Ru ? getExampleFirstInitialValue(Language.Ru) : getExampleFirstInitialValue(Language.En);
   const exampleSecondInitialValue =
-    languageMode === Language.Ru ? exampleSecondInitialValueRu : exampleSecondInitialValueEn;
+    languageMode === Language.Ru
+      ? getExampleSecondInitialValue(Language.Ru)
+      : getExampleSecondInitialValue(Language.En);
   const exampleThirdInitialValue =
-    languageMode === Language.Ru ? exampleThirdInitialValueRu : exampleThirdInitialValueEn;
+    languageMode === Language.Ru ? getExampleThirdInitialValue(Language.Ru) : getExampleThirdInitialValue(Language.En);
```

Comment:
Can it be simplified even more? Smth like:
const exampleFirstInitialValue = getExampleFirstInitialValue(languageMode) and check languageMode inside getExampleFirstInitialValue